jest.config.ts

```diff
@@ -3,9 +3,15 @@
 
 import { Config } from "jest";
 import type { JestConfigWithTsJest } from "ts-jest";
+import { createDefaultEsmPreset } from "ts-jest";
+
+const defaultEsmPreset = createDefaultEsmPreset();
 
 const baseConfig: Config = {
-    preset: "ts-jest",
+    ...defaultEsmPreset,
```

Comment:
These are changes to support esm in jest, copied from their docs (seems to work ok).